tq/transfer_queue.go

```diff
@@ -506,6 +522,19 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 	next := q.makeBatch()
 	tracerx.Printf("tq: sending batch of size %d", len(batch))
 
+	enqueueRetry := func(t *objectTuple, err error) {
+		count := q.rc.Increment(t.Oid)
+		t.ReadyTime = q.rc.ReadyTime(t.Oid)
+		delay := time.Until(t.ReadyTime).Seconds()
+
+		var errMsg string
+		if err != nil {
+			errMsg = ": " + err.Error()
```

Comment:
We usually would prefer to write this as `fmt.Sprintf(": %s", err)`.  That's slightly more efficient as well as more common in our codebase.